tox/__init__.py

```diff
@@ -1,89 +1,29 @@
-import os
-import signal
-
+import pluggy
 from pkg_resources import DistributionNotFound
 from pkg_resources import get_distribution
 
-from .hookspecs import hookimpl
+from .exc import exception  # noqa
```

Comment:
why is the noqa needed here?

Reply:
dunno, might not be necessary at all, but given that I will do some work on exception now I'll check.